src/components/SwapWidget/LimitOrder/index.tsx

```diff
@@ -163,6 +177,54 @@ const LimitOrder: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisibl
     [onUserInput],
   );
 
+  // setting default tokens
+  const filterCurrencies = (symbolName: string): Currency[] => {
+    const tokens = Object.values(allTokens);
+    tokens.unshift(CAVAX[chainId] as Token);
+    const _tokens = tokens.filter((token) => token !== CAVAX[chainId]);
+    const currencies = CHAINS[chainId]?.evm ? [CAVAX[chainId], ..._tokens] : _tokens;
+    return currencies.filter((token) => token?.symbol?.toLowerCase() === symbolName.toLowerCase());
+  };
```

Comment:
Once you change symbol to address based on my first comment, then you don't need this function as you can get token object using `useTokenHook` mapping. then you can convert token to currency using `unwrappedToken` method.